src/main/java/com/iota/iri/conf/BaseIotaConfig.java

```diff
@@ -812,7 +812,7 @@ protected void setCacheSizeBytes(int cacheSizeBytes) {
 
     @Override
     public Hash getCoordinator() {
-        return Defaults.COORDINATOR_ADDRESS;
+        return Defaults.COORDINATOR;
```

Comment:
The reason I changed it now is because of Alon's comment: https://github.com/iotaledger/iri/pull/1159#discussion_r234547241

Now I can either:
a) Disregard Alon's comment and have a discrepancy between the config name and the variable. But I think Alon was right so I'd rather not do that
b) Change the method name to be `getCoordinatorAddress` like you offered.

(b) sounds correct to me BUT:
1. This will change the configuration name in the INI file (and we also need to change the name of the flag to match). Changing the configuration names themselves shouldn't be in the context of this PR, which is just about documentation.
2. Changing config name requires both our devops and the community to change their settings and various scripts, thus inducing more work. So at the very least, an issue must be created on Github to make sure we don't miss on communicating (happened before).
By the way, if we change our configurations again, maybe we want to bundle in more changes for the same release (like https://github.com/iotaledger/iri/issues/1221) so people won't have to maintain configurations for each release. 
3. If we change the config name, note that people will have to start typing a longer name (COORDINATOR_ADDRESS, so many characters). You can argue that clarity might be more important. I would answer that I don't have really strong feelings on this last point, but I still raised it.


To summarize this long comment:
If you want this change, feel free to open an issue :-)